### ISSUE TITLE
Support --propver flag in import command 

### ISSUE THREAD

**Issue by vreddhi** [akamai/cli-promotional-deployment#73]
It is helpful if we can support the version number in import command of property-manager module.

Additional reference:
https://community.akamai.com/customers/s/question/0D54R00008GP5pO/import-the-latest-prod-activate-version-using-the-cli?language=en_US

**Comment by nick-s-morgan** [comment]
I agree this would be very useful. An example scenario is where a Property has been de-activated and we want to take a baseline version from the latest version in Production.
